docs/components/miranum-connect/java-client/miranum-worker.md

```diff
@@ -112,6 +112,19 @@ used as input in a subsequent activity of our process.
 For an actual example take a look on our [miranum-connect guide](../../../guides/getting-started/pizza-order-miranum.md).
 :::
 
+### Retry mechanism
+If a job fails the worker can perform retries. Be aware that the behavior differs between Camunda Platform 7 & 8. 
+In Camunda Platform 8 the retries are set in the BPMN file. Camunda 7, does not provide the functionality to set 
+retries in the BPMN file. Hence, we developed two different approaches to set the retries in Miranum-Connect.
```

Comment:
Do we write miranum connect always like this "Miranum-Connect"?

*Hence, we developed two different approaches to set the retries in Miranum-Connect for Camunda 7.